docs/assets/style.css

```
@@ -1,7 +1,10 @@
-.md-header {
+.md-header, .md-sidebar--primary {
     /* pull z-index up to be above the codemirror gutters (z-index: 3).
      * original value is z-index: 2 */
-    z-index: 10;
+    z-index: 20;
+}
+.md-overly {
+  z-index: 15;
```

Typo + mismatch in indentation

```suggestion
.md-overlay {
    z-index: 15;
```